tests/NLog.Database.Tests/DatabaseTargetTests.cs

```diff
@@ -868,7 +868,7 @@ public void ParameterDbTypePropertyNameTest()
 Parameter #1 Direction=Input
 Parameter #1 Name=@level
 Parameter #1 MockDbType=Int32
-Parameter #1 Value=""{0}""
+Parameter #1 Value={0}
```

Comment:
Now `dbType=' MockDbType.int32  '` is correctly interpreted and the value is rendered as an integer instead of a string; hence no more double quotes.